modules/draco/src/index.ts

```diff
@@ -14,6 +15,30 @@ export type {DracoMesh, DracoLoaderData};
 export type {DracoWriterOptions} from './draco-writer';
 export {DracoWriter} from './draco-writer';
 
+/**
+ * Browser worker doesn't work because of issue during "draco_encoder.js" loading.
+ * Refused to execute script from 'https://raw.githubusercontent.com/google/draco/1.4.1/javascript/draco_encoder.js' because its MIME type ('') is not executable.
+ */
+export const DracoWriterWorker = {
+  id: 'draco-writer',
```

Comment:
Can we get away with only defining one DracoWriter, and hide the platform dependent parts inside:

```suggestion
  id: isBrowser ? 'draco-writer' : 'draco-writer-nodejs`,
```

Reply:
Done